content/en/apps/guides/performance/telemetry.md

```diff
@@ -97,4 +97,22 @@ When the aggregate doc is created the Telemetry service also includes a snapshot
 | `dbInfo.idb_attachment_format` | The format of database attachments. |
 | `dbInfo.db_name` | The name of the local database. |
 | `dbInfo.auto_compaction` | Whether or not auto compaction is set. |
-| `dbInfo.adapter` | The database adapter being used. |
\ No newline at end of file
+| `dbInfo.adapter` | The database adapter being used. |
+
+## Export data to JSON
+
+{{% alert title="Note" %}}
+Telemetry data can be viewed directly in your browser with [Fauxton](https://couchdb.apache.org/fauxton-visual-guide/) at `https://{{CHT_INSTANCE_URL}}/_utils`, and navigating the `medic-users-meta` database.
+{{% /alert %}}
+Follow the steps below to export all telemetry in JSON for further analysis or visualization:
+
+1. Open a command line terminal in the directory where you want to save the export files
+1. Ensure all that both [`node`](https://nodejs.org/en/) and [`npm`](https://www.npmjs.com/get-npm) are installed and that the needed `node` libraries are installed: `npm install inquirer pouchdb-core fs path minimist pouchdb-adapter-http`
+1. Get a current copy of the script: `curl -s -o get_users_meta_docs.js https://raw.githubusercontent.com/medic/cht-core/master/scripts/get_users_meta_docs.js` 
+
+Then to export the data run this command, replacing `USERNAME`, `PASSWORD`, `COUCH_SERVER` and `COUCH_PORT`  with your own values: `node get_users_meta_docs.js --mode batch --type telemetry https://USERNAME:PASSWORD@COUCH_DB_SERVER:COUCH_PORT/medic-users-meta > telemetry.json`. 
+
+For example, if your user was `admin`, your password was `pass`, your server was `localhost` and your port was `5984`, you would run `bash
+node get_users_meta_docs.js --mode batch --type telemetry https://admin:pass@localhost:5984/medic-users-meta > telemetry.json`
```

Comment:
Keep in present if possible. Also, you use `bash` here, but not in the steps above... is it needed?

```suggestion
Keep in present if possible and code line on its own line. Also, you use `bash` here, but not in the steps above... is it needed?

```suggestion
For example, if your user is `admin`, your password is `pass`, your server is `localhost`, and your port is `5984`, you would run the following:
`node get_users_meta_docs.js --mode batch --type telemetry https://admin:pass@localhost:5984/medic-users-meta > telemetry.json`
```